mrp_brewing/report/report_brew_register.xml

```diff
@@ -1,181 +1,181 @@
-	                                    </t>
-                                    	<t t-if="lign_count != 0">
-		                                    <tr>
-			                                   	<td/>
-			                                   	<td/>
-			                                	<td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td>
-			                                        <span t-esc="stm.product_id.name"/>
-			                                    </td>
-			                                    <td>
-			                                        <span t-esc="stm.product_uom_qty"/>
-			                                    </td>
-			                                    <td>
-			                                        <span t-esc="stm.product_uom.name"/>
-			                                    </td>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-			                                    <td/>
-		                                    </tr>
-		                            	</t>
-		                            	<t t-set="lign_count" t-value="1"/>
+    <template id="mrp_brewing.brew_register">
+        <t t-call="report.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="report.internal_layout">
+                    <div class="page">
+                        <h2 style="text-align:center">Brew Register Report</h2>
+                        <br/>
+                        <h3 style="text-align:center">Authorized warehousekeeper n°</h3>
+                        <!-- <div class="row mt32 mb32">
+                            <div class="col-xs-4">
+                                <strong>Target Moves:</strong>
+                                <p>
+                                    <span t-if="data['target_move'] == 'all'">All Entries</span>
+                                    <span t-if="data['target_move'] == 'posted'">All Posted Entries</span>
+                                </p>
+                            </div>
+                            <div class="col-xs-4">
+                                <p>
+                                    <t t-if="data['date_from']"><strong>Date from :</strong> <span t-esc="data['date_from']"/><br/></t>
+                                    <t t-if="data['date_to']"><strong>Date to :</strong> <span t-esc="data['date_to']"/></t>
+                                </p>
+                            </div>
+                        </div> -->

```

Comment:
@vvrossem as you're on it better remove commented code

Reply:
done

Reply:
👍 let's merge now ;)